web/services/clusters.go

```diff
@@ -49,6 +49,28 @@ func NewClustersService(db *gorm.DB, checksService ChecksService) *clustersServi
 
 func (s *clustersService) GetAll(filter *ClustersFilter, page *Page) (models.ClusterList, error) {
 	var clusters []entities.Cluster
+	var healthFilteredClusters []string
+
+	// Filter the clusters by Health
+	if filter != nil && len(filter.Health) > 0 {
+		var checksResults []entities.ChecksResult
```

Comment:
Yes, i had the same internal discussion. It could go in `checks_result` service as well, but I didn't want to mix the filtering even there. But I will have a look in any case!
Maybe I can wrap the SQL query in that file at least

Reply:
Done in https://github.com/trento-project/trento/pull/725/commits/1c80b6e7e9221b72f642dcc9e8ab23a88fe17260